pkg/littlefs/lfs_log.h

```diff
@@ -0,0 +1,22 @@
+#include "log.h"
```

Comment:
As CI says, the two `lfs_log.h` files need the license header and the c++ header guard.

Reply:
I will fix this.

I suppose also the macros need documented as well, or can that be omitted since they are just a re-implementations of the pkg provided ones?

Reply:
Not sure, but I'd guess the CI will ask for documentation.

Reply:
fixed

Reply:
Thanks! Unfortunately, CI is still not fully happy...

Reply:
Hopefully I got it this time. I also fixed some white space typos and coding convention errors.